src/sparql/keywords.ts

```diff
@@ -1,626 +1,152 @@
-// @ts-ignore: import types for declarations
-import { createToken, TokenType } from 'chevrotain';
-
-const MAX_LENGTH = createToken({
+import { ITokenConfig } from 'chevrotain';
+import helper from 'helpers/chevrotain';
+
+export const createKeyword = ({
+  longer_alt = UNKNOWN,
```

Comment:
Why not set the default of the `longer_alt` in the helpers? Would that make it possible to import `createKeyword` directly from the helpers for the SMS tokens rather than from here?

Reply:
looks like I explain this on an earlier comment => `src/sms/tokens.ts`